.github/workflows/container-image.yml

```diff
@@ -37,176 +35,174 @@ jobs:
       - name: Lint git commit messages
         run: make check-gitlint
 
+
+
+
+    strategy:
+      matrix:
+        os: [centos, fedora]
+        arch: [amd64, x86_64]
```

Comment:
I don't know if you meant to do this as an experiment or it's a mistake. `amd64` and `x86_64` are the same architecture. One's the "golang" style of name and the other is the "kernel" style of name. I have no idea if trying `arm64` works on the CI infra yet, but I was wondering if you meant to put it in the list.